packages/core/__tests__/Resource/Decorator/Resource/Resource.relationshipsDecorator.test.js

```diff
@@ -0,0 +1,65 @@
+import Resource from '../../../../src/Resource/Resource';
+import relationshipsDecorator
+  from '../../../../src/Resource/Decorator/Resource/relationshipsDecorator';
+
+describe('The relationships decorator for a resource', () => {
+  test('that the relationships are correctly set from the decorator configuration', () => {
+    Resource.decorators.push(relationshipsDecorator.create({
```

Comment:
The relationship decorator is different for each resource I assume?
Is this code not adding the decorator to the general resource object? If that's the case, the code could end up with to many decorators on the resource. Would it be best practice to add the decorator to the created resource?
In that case this should also be done in this test.

Reply:
There is 1 relationships decorator which defines all the relations for each resource, so only 1 decorator for all resources. I created it this way so you can define your relationships once and then the Resource.create will use the same definition each time.

Reply:
I documented the usage in the feature/documentation PR.

Reply:
Documentation is indeed needed. Most documentation is read badly however.
Also the problem may arise if you need to connect to more than one backend: in that case the relationships most certainly will differ. Solving this issue without introducing undesirable coupling is not trivial however.

Reply:
Discussed irl and explained to satisfaction. Multiple connectors/backends is not relevant to the relationships as you can define all relationships per resource type for the decorator, regardless of the backend they are defined on.

Reply:
After a conversation I must conclude that I was wrong. I think it will work like it is :-)